src/lib/runner.js

```diff
@@ -63,30 +70,70 @@ class Runner {
         stdio: this.enableStdOut ? 'inherit' : null
       });
 
-      this.childProcess.on('close', code => {
-        if (err !== '' && code && code !== 0) {
```

Comment:
removed this `err` bit because it wasn't actually being used. It's written in the stderr hook, which also rejects. So if err is not empty here, that means reject has already been called.

Reply:
Reverting all this for now. I'll bring it up in a separate PR